docs/standard/components.md

```diff
@@ -54,27 +53,34 @@ Mono is the runtime used by Xamarin apps.  It implements the .NET Standard, whic
 
 To learn more about Mono, see the [Mono documentation](http://www.mono-project.com/docs/).
 
+## .NET runtimes
+
+A runtime is the execution environment for a managed program. The OS is part of the runtime environment but is not part of the .NET runtime. Here are some examples of .NET runtimes:
+ 
+ - Common Language Runtime (CLR) for .NET Framework
+ - Core Common Language Runtime (CoreCLR) for .NET Core
+ - .NET Native for Universal Windows Platform 
+
 ## .NET tooling and common infrastructure
 
-Tooling for .NET is also common across each implementation of .NET.  These include, but are not limited to:
+Some tooling for .NET is common across all implementations of .NET.  These include, but are not limited to:
 
-* The .NET languages and their compilers
-* Runtime components, such as the JIT and Garbage Collector
-* .NET project system (sometimes known as "csproj", "vbproj", or "fsproj")
-* MSBuild, the build engine used to build projects
-* NuGet, Microsoft's package manager for .NET
-* The .NET Core CLI, a cross-platform command-line interface for building .NET projects
-* Open Source build orchestration tools, such as CAKE and FAKE
+- The .NET languages and their compilers
+- .NET project system (sometimes known as "csproj", "vbproj", or "fsproj")
+- MSBuild, the build engine used to build projects
+- NuGet, Microsoft's package manager for .NET
+- Open Source build orchestration tools, such as CAKE and FAKE
```

Comment:
There are some opportunities here to cross-link if u want to ...

MSBuild :point_right: \[MSBuild](/visualstudio/msbuild/msbuild)
NuGet :point_right: \[NuGet](/nuget/)
CAKE :point_right: \[CAKE](http://cakebuild.net/)
FAKE :point_right: \[FAKE](https://fake.build/)